### PR TITLE
Update end-to-end walkthrough for JupyterCon 2023

### PR DESCRIPTION
**Pull request by seeM**
These are some updates in preparation for the nbdev tutorial at JupyterCon next week. I also made a tiny fix to the CSS due to upstream changes in Quarto's styling.

The main change is to suggest Lab instead of classic Notebook, which I would now personally recommend for the following reasons:

- Growing extension ecosystem:
    - I can't live without the wonderful [`jupyterlab-quarto`](https://github.com/quarto-dev/quarto/tree/main/apps/jupyterlab) these days. It's still under development but already renders a bunch of things in your notebook like Quarto frontmatter, callouts, and fenced divs.
    - Interesting work being done on LLM-related extensions e.g. [`jupyter-ai`](https://github.com/jupyterlab/jupyter-ai) which adds an LLM chat interface to the sidebar and a magic command.
    - [`jupyterlab-git`](https://github.com/jupyterlab/jupyterlab-git) for notebook diffs.
- Version 4.0.0 release candidate was released last week, which includes lots of improvements including built-in functionality from the classic collapsible headings extension (including keyboard shortcuts). However, it is worth noting that extensions will need to migrate to v4 and I'm not sure what this process will look like!

I considered recommending VSCode instead, but after trying it out for a month for the second time, I personally encountered too many issues with performance (terrible fps) and robustness (the kernel often gets into an irrecoverable state), both of which were experienced by other people I spoke to. These issues seem not to affect everyone, but I couldn't figure out how to fix them or how to get support -- very frustrating!

### CODE REVIEW

File: nbs/tutorials/tutorial.ipynb [hunk 1]
```diff
@@ -64,30 +64,32 @@
     "4. nbdev\n",
```
Comment (hamelsmu) [inline nbs/tutorials/tutorial.ipynb:64]: <p>Conda has a <code>-y</code> flag</p><p>Looks like pip install jupyterlab will not ask (will only ask for uninstall, in which case you can pass <code>-y</code> there  too</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (hamelsmu) [inline nbs/tutorials/tutorial.ipynb:64]: <p>nit: Jupyter Lab should be "JupyterLab" (one word)</p><p> </p><p>reference: https://jupyter.org/try</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (hamelsmu) [inline nbs/tutorials/tutorial.ipynb:64]: <p>Pass the <code>-y</code> flag here too? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

**Review by hamelsmu — APPROVED** [review]
Looks good, suggested some minor changes.

**Comment by hamelsmu** [comment]
@seeM when you are done here you might want to update this site https://github.com/fastai/jupytercon-2023 with a link to this tutorial.  (Sharing here for visibility)

**Comment by jph00** [comment]
Thanks @seeM lgtm! :D

File: nbs/tutorials/tutorial.ipynb (hunk 1 above)
Comment (seeM) [reply nbs/tutorials/tutorial.ipynb:64]: <p>Good to know, will update</p>

Comment (seeM) [reply nbs/tutorials/tutorial.ipynb:64]: <p>Ah good catch, will update</p>

Comment (seeM) [reply nbs/tutorials/tutorial.ipynb:64]: <p>Done</p>

**Pull request merged**